### ISSUE TITLE
Need to load the image inside canvas without chopping off 

### ISSUE THREAD

**Issue by eshwerintouch** [webfactorymk/ng2-canvas-whiteboard#37]
Hi team,

The library was very nice. But I see that the original image was chopped off on top and bottom.

I mean that the image width was completely fine but it was not covering the all content when it comes to height. Some of the contents on top and bottom of the image was deleted when it was on edit mode.

Please fix it OR let me know if there's any option to load it completely.

**Comment by eshwerintouch** [comment]
Even after giving AspectRatio = 0 the canvas still cut of some part of the image.

**Comment by Fridthjof** [comment]
Did you find somewhat of a solution? 

i currently got the same issue.

**Comment by eshwerintouch** [comment]
@Fridthjof Nothing particularly. But I found out that if we give the container height and width same as the original image aspect ratio it loads up fine. As the high resolution images go out of the window if I try to give the same height dynamically by getting the original height now I do resize the image using [ngx-pic](https://www.npmjs.com/package/ngx-pica) as per my container width and load the image so I get the exact image.

**Comment by eshwerintouch** [comment]
@Fridthjof reference [link](https://pages.onepgr.com/pages/95061?passcode=453463) where I have used the editor

**Comment by Fridthjof** [comment]
I Fixed it by dynamically setting aspectRatio, by dividing the uploaded image height and width and inserting that value into the aspectRatio option.

**Comment by eshwerintouch** [comment]
Yes we could do that but in my case I had the zoom event too in edit mode.
But it was not showing completely if I create that way so. Can you please
share the snippet incase if that works for my scenario too

**Comment by ashishvspl076** [comment]
@eshwerintouch  @Fridthjof  have u solved image chopping issue ? i am also facing this issue
